backend/validators/did_validator.py

```
@@ -6,7 +6,8 @@
 import logging
 from typing import Dict, Optional, Any
 
-from core_foundation_enhanced import logger
+import logging
```

Ruff (0.12.2)

Redefinition of unused `logging` from line 6

Remove definition: `logging`

(F811)